tests/foreman/ui/test_location.py

```diff
@@ -565,8 +577,13 @@ def test_positive_remove_subnet(self):
                         mask='255.255.255.0',
                     ).create()
                     self.assertEqual(subnet.name, subnet_name)
-                    set_context(session, org=ANY_CONTEXT['org'])
-                    make_loc(session, name=loc_name, subnets=[subnet_name])
+                    set_context(session, org=self.org_.name)
```

Comment:
@sthirugn @abalakh we can simplify that turning **self.session_user** and **self.session_org** in to properties, so in subclasses like this we only need to override the properties, it will make it simpler.

```python
class LocationTestCase(UITestCase):
     
     @property
     def session_org(self):
         return entities.Organization(...).create() 
``` 

So the above will make the session user to get that organization_id by default instead of "1"